src/featurize.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+import scipy as sp
+import sklearn
+import sklearn.preprocessing
+import copy
```

Comment:
see linter, also importing sklearn and sklearn.preprocessing might be redundant for this module. Maybe choose one or the other?

Reply:
You are correct. I only need sklearn.preprocessing.